js/src/dashboard/summary-section/index.js

```diff
@@ -106,17 +107,27 @@ const PaidPerformanceCard = () => {
 };
 
 export default function SummarySection() {
+	const { loaded, data: adsCampaignsData } = useAdsCampaigns();
 	const { adsSetupComplete } = glaData;
+	let showCampaignPromotionCard = true;
+	if ( adsSetupComplete ) {
+		if ( ! loaded ) {
+			return null;
+		}
+		if ( adsCampaignsData?.length > 0 ) {
+			showCampaignPromotionCard = false;
+		}
+	}
```

Comment:
Since we reverted the change to useAdsCampaigns() and no longer need to check for the `adsSetupComplete` status, this can be further simplified to simply check for the length of `adsCampaignsData` (returning `null` until the campaign data is loaded is a good idea!)

```suggestion
	if ( ! loaded ) {
		return null;
	}

	const showCampaignPromotionCard = Boolean( ! adsCampaignsData?.length );
```